packages/compass-components/package.json

```diff
@@ -63,6 +63,7 @@
     "@leafygreen-ui/search-input": "^4.0.0",
     "@leafygreen-ui/segmented-control": "^9.0.0",
     "@leafygreen-ui/select": "^13.0.0",
+    "@leafygreen-ui/split-button": "../../../leafygreen-ui/packages/split-button/leafygreen-ui-split-button-v3.0.0.tgz",
```

Comment:
This is a temporary workaround and needs to be updated once my proposed changes merge and release.